### PR TITLE
refactor(events): convert to ts

### PR DESCRIPTION
**Pull request by naorpeled**

### CODE REVIEW

File: testUtils/testdata/events/appsync-event.json [hunk 1]
```diff
@@ -1,40 +1,35 @@
...
-    "info": {
-      "fieldName": "bundles",
-      "parentTypeName": "Query",
-      "variables": {
-
-      }
-    },
-    "stash": {
-
+  "arguments": {
```
Comment (naorpeled) [inline testUtils/testdata/events/appsync-event.json:2]: removed "context",
it doesn't appear in the `aws-lambda` type definitions

File: src/events/event-data.types.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  messageId: AppSyncResolverEventHeaders['x-amzn-trace-id'];
+}
+
+export interface ApiGatewayV1EventData {
+  messageId: APIGatewayProxyEvent['requestContext']['requestId'];
+  httpMethod: APIGatewayProxyEvent['httpMethod'];
+  resource: APIGatewayProxyEvent['resource'];
+  stage: APIGatewayProxyEvent['requestContext']['stage'];
+  api: APIGatewayProxyEvent['headers'][0];
+}
```
Comment (doriaviram) [inline src/events/event-data.types.ts:77]: 💪

**Review by doriaviram — COMMENTED** [review]
Lets talk offline about the package-lock changes

**Review by OrrLumigo — APPROVED** [review]

**Review by doriaviram — APPROVED** [review]

**Pull request merged**